addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -6,6 +6,12 @@
     @include o-text-overflow;
 }
 
+.dropdown-menu {
+    max-height: 70vh;
+    overflow: auto;
+    background-clip: border-box;
+}
+
```

Comment:
What is the need for this backport? Don't we already set the max-height in javascript?

Reply:
> What is the need for this backport? Don't we already set the max-height in javascript?

I updated the PR message with some explanations.
the dropdown's height need to be limited for UX

Reply:
Isn't it a bit risky for stable though? Especially since we have javascript handling and limiting its height. I guess I'm just wondering why wasn't this back-ported to 15.0 to begin with if the goal was to impact all of Odoo's UI.

Reply:
> Isn't it a bit risky for stable though? Especially since we have javascript handling and limiting its height. I guess I'm just wondering why wasn't this back-ported to 15.0 to begin with if the goal was to impact all of Odoo's UI.

Actually, it was a request from a pair-review ... At first, I hadn't added these lines (backport) ...
So I don't care about deleting them, it's not a big deal to leave it like that IMO

Reply:
I think it's fine in 16.4+ but let's not risk breaking customisation in 15.0, it's always been the way it is as of now right? Or is there a commit that removed those lines wrongly (in 15.0)?

Reply:
> I think it's fine in 16.4+ but let's not risk breaking customisation in 15.0

ok :+1: 

> Or is there a commit that removed those lines wrongly (in 15.0)?

Maybe [this one](https://github.com/odoo/odoo/commit/84715436d87bb05b421bc9ccaacda67d07571690)

Reply:
@detrouxdev I made the changes